FWCore/PythonParameterSet/python/testTask_cff.py

```diff
@@ -0,0 +1,248 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
As we discussed I will move it to python/test. It still needs to be under python because one of its purposes is to test the behavior of the Process load function and python files in the test directory cannot be loaded normally by design of the way it looks for python files to import.